src/drafts/index.ts

```diff
@@ -10,4 +10,4 @@ export * from '../ActionList2'
 export * from '../Button2'
 export * from '../ActionMenu2'
 export * from '../DropdownMenu2'
-export * from '../Label2'
+export * from '../PageLayout'
```

Comment:
Can we remove this line please? ☝️ .. it's been removed in @colebemis graduation PR.

Reply:
Thanks for catching this. Bad merge on my part.